sortedcontainers/sortedset.pyi

```diff
@@ -0,0 +1,81 @@
+import typing
+from typing import (
+    Any,
+    AbstractSet,
+    Callable,
+    Generic,
+    Hashable,
+    Iterable,
+    Iterator,
+    List,
+    MutableSet,
+    Optional,
+    Sequence,
+    Tuple,
+    Type,
+    Set,
+    TypeVar,
+    Union,
+)
+
+# --- Global
+
+_T = TypeVar("_T", bound=Hashable)
+_S = TypeVar("_S", bound=Hashable)
+_SS = TypeVar("_SS", bound="SortedSet")
+_Key = Callable[[_T], Any]
+
+class SortedSet(MutableSet[_T], Sequence[_T]):
+    def __init__(self, iterable: Optional[Iterable[_T]] = ..., key: Optional[_Key] = ...) -> None: ...
+    @classmethod
+    def _fromset(cls, values: Set[_T], key: Optional[_Key] = ...) -> SortedSet[_T]: ...
+    @property
+    def key(self) -> Optional[_Key]: ...
+    def __contains__(self, value: Any) -> bool: ...
+    @typing.overload
+    def __getitem__(self, index: int) -> _T: ...
+    @typing.overload
+    def __getitem__(self, index: slice) -> List[_T]: ...
+    def __delitem__(self, index: Union[int, slice]) -> None: ...
+    def __eq__(self, other: Any) -> bool: ...
+    def __ne__(self, other: Any) -> bool: ...
+    def __lt__(self, other: Iterable[_T]) -> bool: ...
+    def __gt__(self, other: Iterable[_T]) -> bool: ...
+    def __le__(self, other: Iterable[_T]) -> bool: ...
+    def __ge__(self, other: Iterable[_T]) -> bool: ...
+    def __len__(self) -> int: ...
+    def __iter__(self) -> Iterator[_T]: ...
+    def __reversed__(self) -> Iterator[_T]: ...
+    def add(self, value: _T) -> None: ...
+    _add = add
```

Comment:
This and all the other assigned methods in this file should be inlined like what was done in `SortedDict`. Some of the dunder methods below will need `# type: ignore`.